gamestonk_terminal/portfolio_optimization/port_opt_api.py

```diff
@@ -0,0 +1,360 @@
+""" Portfolio Optimization Functions"""
```

Comment:
You can call this `optimizer_view` instead, and `optimizer_helper` to the other one to follow same naming convention